ADAL/src/telemetry/ADTelemetryAPIEvent.h

```diff
@@ -30,6 +30,7 @@
 - (void)setExtendedExpiresOnSetting:(NSString*)extendedExpiresOnSetting;
 - (void)setPromptBehavior:(ADPromptBehavior)promptBehavior;
 - (void)setUserInformation:(ADUserInformation*)userInfo;
+- (void)setUserId:(NSString*)userId;
```

Comment:
Sure, I will fix it :)

I notice there is an inconsistency for this issue in our code.
Probably we may need to go through the code in the future and make it consistent.